src/core/Database.cpp

```diff
@@ -34,9 +33,11 @@ Database::Database()
     , m_emitModified(false)
     , m_uuid(Uuid::random())
 {
-    m_data.cipher = KeePass2::CIPHER_AES;
+    m_data.cipher = KeePass2::CIPHER_CHACHA20;
```

Comment:
Yes. We may (and probably will) change that in the future, but for now we should default to the proved, stable and non-experimental ciphers.